src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php

```diff
@@ -186,29 +195,72 @@ public function iConfirmMyOrder(): void
      */
     public function iCompleteTheAddressingStep(): void
     {
+        if (!array_key_exists('email', $this->content)) {
+             $this->iSpecifyTheEmailAs('customer@example.com');
+        }
```

Comment:
IMO it should be reverted, as there already is setting customer as null in https://github.com/Sylius/Sylius/blob/master/src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php#L229:L231